src/bundler.rs

```diff
@@ -8,65 +8,65 @@ pub struct JavascriptBundle {
     pub content: String,
 }
 
-pub fn bundle(entry_module: String, module_map: HashMap<String, lexer::JavascriptModule>) -> JavascriptBundle {
-  let mut content = String::new();
-  content.push_str("import { insertModule, createModuleUrl, resolveImportSpecifier } from \"/bloom.js\";\n");
+pub fn bundle(entry_module: String, module_map: HashMap<String, parser::JavascriptModule>) -> JavascriptBundle {
+    let mut content = String::new();
```

Comment:
Hmm maybe would be best to introduce a git hook using rustfmt so that we don't have to worry about styling/indentation thrashing. https://github.com/rust-lang/rustfmt